lib/index.js

```diff
@@ -8,6 +8,7 @@ var Spawn = require("child_process").spawn
   , Ul = require("ul")
   , OneByOne = require("one-by-one")
   , Sliced = require("sliced")
+  , Q = require('q')
```

Comment:
TBH I have never seen that approach used in any other library. In my opinion overhead of including and creating promise is really extremely small and it is better to keep API cleaner and simpler. Please note that it is not breaking current interface. What do you think? :innocent: